src/cpp/CFFICommon.ml

```diff
@@ -0,0 +1,52 @@
+(*
```

Comment:
Looks like this module lacks its interface (`mli`) file

Reply:
Added